examples/react-native-example/src/App.js

```diff
@@ -10,12 +15,18 @@ export default function App() {
       host={process.env.EXPO_PUBLIC_KNOCK_HOST}
       userId={process.env.EXPO_PUBLIC_KNOCK_USER_ID}
     >
-      <KnockFeedProvider feedId={process.env.EXPO_PUBLIC_KNOCK_FEED_CHANNEL_ID}>
-        <View style={styles.container}>
-          <StatusBar style="auto" />
-          <NotificationFeed />
-        </View>
-      </KnockFeedProvider>
+      <ExpoPushNotificationProvider
```

Comment:
If this provider is using the `useKnockClient`, etc. hooks, it'll need to be a child of `KnockFeedProvider` so it has access to the context.